src/services/latex.js

```diff
@@ -97,19 +97,73 @@ var latexMathParser = (function() {
   latexMath.latexText = mathMode.or(textChar).many();
 
   latexMath.parse =
-  latexMath.latexText.parse = Parser.p._parse;
+  latexMath.latexText.parse = function(latex, latexCmds) {
+    var parser = this;
+    return localLatexCmds.let(latexCmds, function() {
+      return parser._parse(latex);
+    });
+  };
+  var localLatexCmds = ENV_VAR(); // LaTeX commands during this parse
 
   return latexMath;
 })();
 
+/**
+ * Concise JSON Schema for `customSymbols` declarations:
+ *   {
+ *     "/[a-z]+/i": {
+ *       spacing: { $enum: ['ord', 'bin', 'rel'] },
+ *       $optional_italic: { $value: true } // only allowed if spacing === 'ord'
+ *     }
+ *   }
+ * (More about Concise JSON Schemas: https://git.io/vwI9k )
+ *
+ * This is based on two classifications TeX uses for typesetting:
+ *   - the type of atom ("ord, op, bin, rel, open, close, punct and inner")
+ *       + determines spacing and line-breaking
+ *   - the family of font (normal upright roman, italic, etc)
+ *
+ * https://github.com/Khan/KaTeX/wiki/Examining-TeX#group-types
+ * http://tex.stackexchange.com/a/38984
+ */
```

Comment:
Point being: looked at MathML's classifications for atoms; decided they shouldn't influence MathQuill API's classifications.